packages/cli/src/commands/node/synced.test.ts

```diff
@@ -0,0 +1,72 @@
+import { viem_testWithAnvilL2 } from '@celo/dev-utils/lib/viem/anvil-test'
+import { stripAnsiCodesFromNestedArray, testLocallyWithViemNode } from '../../test-utils/cliUtils'
+import Synced from './synced'
+
+let nodeIsSynced = jest.fn()
+let nodeIsSyncedRaw = jest.fn()
+jest.mock('../../utils/helpers', () => ({
+  ...jest.requireActual('../../utils/helpers'),
+  nodeIsSyncedRaw: () => nodeIsSyncedRaw(),
+  nodeIsSynced: () => nodeIsSynced(),
+}))
+
+viem_testWithAnvilL2('node:synced cmd', (client) => {
+  beforeEach(() => {
+    process.env.NO_SYNCCHECK = undefined
```

Comment:
minor: what about `delete process.env.NO_SYNCCHECK`? I think it's just more descriptive syntactically speaking